src/components/ebay-combobox/component.js

```diff
@@ -62,37 +70,37 @@ export default {
     },
 
     handleComboboxClick(e) {
-        if (e.target === document.activeElement && this.expander && !this.isExpanded()) {
-            this.expander.expanded = true;
+        if (e.target === document.activeElement) {
+            this.expand();
         }
     },
 
     handleComboboxKeyDown(originalEvent) {
         eventUtils.handleUpDownArrowsKeydown(originalEvent, () => {
             originalEvent.preventDefault();
 
-            if (this.expander && !this.expander.expanded) {
+            if (!this.isExpanded()) {
                 this.activeDescendant.reset();
-                this.expander.expanded = true;
+                this.expand();
             }
         });
 
         eventUtils.handleEnterKeydown(originalEvent, () => {
-            if (this.expander.expanded) {
+            if (this.isExpanded()) {
                 const selectedIndex = this.activeDescendant.index;
 
                 if (selectedIndex !== -1) {
                     this._setSelectedText(this._getVisibleOptions()[selectedIndex].text);
                 }
 
                 if (this.input.expanded !== true) {
-                    this.expander.expanded = false;
+                    this.collapse();
                 }
```

Comment:
A little tangential to this PR: I think that the parts that touch `input.expanded` should have adequate comments why they are necessary.